lib/jelix-admin-modules/jauthdb_admin/controllers/default.classic.php

```diff
@@ -413,26 +448,44 @@ function delete(){
             jMessage::add(jLocale::get('crud.message.delete.invalid.pwd'), 'error');
             $rep->action = 'default:confirmdelete';
             $rep->params['j_user_login'] = $login;
+
             return $rep;
         }
 
-        if( $login !== null && jAuth::getUserSession()->login != $login){
-            if(jAuth::removeUser($login)) {
+        if ($login !== null && jAuth::getUserSession()->login != $login) {
+            if (jAuth::removeUser($login)) {
                 jMessage::add(jLocale::get('crud.message.delete.ok', $login), 'notice');
                 $rep->action = 'default:index';
-            }
-            else{
+            } else {
                 jMessage::add(jLocale::get('crud.message.delete.notok'), 'error');
                 $rep->action = 'default:view';
                 $rep->params['j_user_login'] = $login;
             }
-        }
-        else {
+        } else {
             jMessage::add(jLocale::get('crud.message.delete.notok'), 'error');
             $rep->action = 'default:index';
         }
+
         return $rep;
     }
 
-}
+    public function autocomplete()
+    {
+        $rep = $this->getResponse('text');
```

Comment:
There is a JSON response object, it's better because it encodes content for you, and return the right mime type. so, indicate 'json'. See jResponseJson to know about its API.